x/ccv/provider/keeper/relay.go

```diff
@@ -267,6 +278,11 @@ func (k Keeper) EndBlockCIS(ctx sdk.Context) {
 	// - Marshaling and/or store corruption errors.
 	// - Setting invalid slash meter values (see SetSlashMeter).
 	k.CheckForSlashMeterReplenishment(ctx)
+
+	// Reset the vsc matured packet handler counter.
+	// Note this assumes vsc matured packets are only handled in the two methods following this call.
+	k.ResetVSCMaturedHandledThisBlock(ctx)
```

Comment:
All the VSCMaturedPackets are handled in EndBlock, more preciselly, in `EndBlockCIS`. Why do we need to store the number of handled packets in the store? We can just use a variable, right?

Reply:
You're right and esp because this feature will not be around for long 👍

Reply:
See https://github.com/cosmos/interchain-security/commit/cf5c452f53c21b404083cceeb3c1eb0eb7f6eebf